package.json

```diff
@@ -17,6 +17,7 @@
   },
   "devDependencies": {
     "eslint": "^1.0.0",
+    "lodash.assign": "^4.2.0",
```

Comment:
Wouldn't hurt to have all of lodash available. It's not like we're trying optimize on size since this server side JS.